README.md

```diff
@@ -3,9 +3,52 @@
 [![Build Status](https://circleci.com/gh/windmilleng/tilt/tree/master.svg?style=shield)](https://circleci.com/gh/windmilleng/tilt)
 [![GoDoc](https://godoc.org/github.com/windmilleng/tilt?status.svg)](https://godoc.org/github.com/windmilleng/tilt)
 
+
+## Using tilt
+`tilt up <service_name>` starts a service once; `tilt up --watch <service_name>` starts it and watches for changes.
+
+Tilt reads from a Tiltfile. A simple Tiltfile is below:
+```
+def backend():
+  repo = local_git_repo('../backend')
+  img = build_docker_image('Dockerfile', 'companyname/backend', '/go/bin/server')
+  img.add('/go/src/github.com/companyname/backend', repo)
+  img.run('go install github.com/companyname/backend/server')
+  return k8s_service(local_file('backend.yaml'), img)
+```
+
+## Mill
+written in a Mill, a dialect of python.
+
+### Mill Builtins
```

Comment:
fwiw, the `pets` documentation had a more expanded form that listed the type of every argument
https://github.com/windmilleng/pets/blob/master/docs/config.md

Reply:
Adopted.